tools/crm_ticket.c

```diff
@@ -616,7 +617,7 @@ modify_ticket_state(gchar *ticket_id, cib_t *cib, pcmk_scheduler_t *scheduler)
     while (g_hash_table_iter_next(&hash_iter, (gpointer *) & key, (gpointer *) & value)) {
         crm_xml_add(ticket_state_xml, key, value);
 
-        if (pcmk__str_eq(key, "granted", pcmk__str_casei)
+        if (pcmk__str_eq(key, PCMK_XA_GRANTED, pcmk__str_none)
```

Comment:
Let's leave these last two casei for compatibility

Reply:
It's not part of the configuration; it's set only by Pacemaker. The constant is public because `"granted"` appears in API output.

Reply:
Actually, that's a mistake. `"granted"` appears only as a value in API output. So this should be an internal constant, and we should have separate `PCMK_VALUE` constants for `"granted"` and `"revoked"`.